xml/System.Runtime.Loader/AssemblyDependencyResolver.xml

```diff
@@ -14,8 +14,17 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>The <see cref="T:AssemblyDependencyResolver" /> allows a program to resolve assemblies and native libraries to paths based on the dependencies of a given assembly.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Runtime.Loader.AssemblyDependencyResolver> enables application developers to more easily develop a plugin architecture in conjunction with custom <xref:System.Runtime.Loader.AssemblyLoadContext?displayProperty=nameWithType>s to isolate plugins and also enable plugins to load dependencies.
```

Comment:
```suggestion
 The `AssemblyDependencyResolver` class enables application developers to more easily develop a plugin architecture in conjunction with custom <xref:System.Runtime.Loader.AssemblyLoadContext?displayProperty=nameWithType> instances to isolate plugins and also enable plugins to load dependencies.
```
I've fenced AssemblyDependencyResolver instead of using an xref because this creates a self-referencing link in the current publishing system.